src/Controllers/HomeController.cs

```diff
@@ -344,7 +344,7 @@ private void CreateJwtCookieAndAppendToResponse(string identityCookie, int altin
             {
                 Name = "AltinnStudioRuntime",
                 SameSite = SameSiteMode.Lax,
-                HttpOnly = true,
+                HttpOnly = false,
```

Comment:
While I agree that this does not matter from a security perspective in LocalTest, I think we should strive to use the same settings for LocalTest and production. We already have APIs in [LocalTest for retrieving tokens](https://github.com/Altinn/app-localtest/blob/1d23de2ef99d47e9b3f9456392352e209b8f4d73/src/Controllers/HomeController.cs#L184), so adding one to retrieve the cookie value seems reasonable to me.

A better approach might be to add an api in app-lib-dotnet that retrieves a PDF using the same code that we actually use for the real pdf generation also.

Reply:
That should work just fine if you think that is a good idea, I only made this to make it easier for myself to test the local PDF generator without having to use postman, and thought I could share it.

If you think its okay to open up for using the PDF-generator at will through app-lib we should definitely do that instead, since you then also get any custom generator settings for the specific app. Maybe we would want to use something like that for the [Print button component](https://docs.altinn.studio/app/development/ux/components/printbutton/) as well, but I can never get a straight answer on that 😅 Maybe it should contain a watermark or something to distinguish it from an actual submitted PDF-receipt?